src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,476 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['servicebus'] = """
+    type: group
+    short-summary: Manage Azure Service Bus namespace, queue, topic, subscription, rule and geo disaster recovery configuration - alias
+
+    """
+
+helps['servicebus namespace'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Namespace
+
+    """
+
+helps['servicebus namespace authorizationrule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Namespace AuthorizationRule
+
+    """
+
+helps['servicebus queue'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Queue and authorizationrule
+
+    """
+
+helps['servicebus queue authorizationrule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Queue AuthorizationRule
+
+    """
+
+helps['servicebus topic'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Topic and authorizationrule
+
+    """
+
+helps['servicebus topic authorizationrule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Topic AuthorizationRule
+"""
+
+helps['servicebus subscription'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Subscription
+"""
+
+helps['servicebus rule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Rule
+"""
+
+helps['servicebus georecovery-alias'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Geo Disaster Recovery Configuration - Alias
+"""
+
+helps['servicebus namespace exists'] = """
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: check for the availability of mynamespace for the Namespace
+          text: az servicebus namespace exists --name mynamespace
+"""
+
+helps['servicebus namespace create'] = """
+    type: command
+    short-summary: Creates a Service Bus Namespace
+    examples:
+        - name: Create a Service Bus Namespace.
+          text: az servicebus namespace create --resource-group myresourcegroup --name mynamespace --location westus
+           --tags ['tag1' 'value1', 'tag2' 'value2'] --sku-name Standard --sku-tier Standard
```

Comment:
`--tags tag1=value1 tag2=value2`
What you currently have doesn't work.

Reply:
Resolved